.github/workflows/spell-checker.yml

```
@@ -0,0 +1,19 @@
+name: Spell Checker
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  yaml-format:
+    runs-on: ubuntu-latest
+    steps:
+      steps:
```

actionlint (1.7.4)

"steps" section must be sequence node but got mapping node with "!!map" tag

(syntax-check)